src/libstd/sys/windows/backtrace.rs

```diff
@@ -334,7 +337,7 @@ pub fn write(w: &mut Write) -> io::Result<()> {
     let _c = Cleanup { handle: process, SymCleanup: SymCleanup };
 
     // And now that we're done with all the setup, do the stack walking!
-    let mut i = 0;
+    let mut i = -1;
```

Comment:
How come this change was made? Was it to skip the first frame? (if so can you leave a comment?)